From 019930ffad39314605b951d449c5dbc55731dab0 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:38 -0400
Subject: [fs] nfsd: better VERIFY comment

Message-id: <1403560314-32525-37-git-send-email-bfields@redhat.com>
Patchwork-id: 84587
O-Subject: [Patch RHEL7 036/172] nfsd4: better VERIFY comment
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

This confuses me every time.

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 41ae6e714a6c25a9932d32a323e8c87f6bac4037
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4proc.c b/fs/nfsd/nfs4proc.c
index 416ead8..ef76ba6 100644
--- a/fs/nfsd/nfs4proc.c
+++ b/fs/nfsd/nfs4proc.c
@@ -1076,8 +1076,10 @@ _nfsd4_verify(struct svc_rqst *rqstp, struct nfsd4_compound_state *cstate,
         cstate->current_fh.fh_dentry, &p,
         count, verify->ve_bmval,
         rqstp, 0);
-
- /* this means that nfsd4_encode_fattr() ran out of space */
+ /*
+  * If nfsd4_encode_fattr() ran out of space, assume that's because
+  * the attributes are longer (hence different) than those given:
+  */
  if (status == nfserr_resource)
   status = nfserr_not_same;
  if (status)
-- 
1.7.1